src/Symfony/Bundle/DebugBundle/Resources/config/services.php

```diff
@@ -50,7 +50,7 @@
                 service('debug.stopwatch')->ignoreOnInvalid(),
                 service('debug.file_link_formatter')->ignoreOnInvalid(),
                 param('kernel.charset'),
-                service('.virtual_request_stack'),
+                service('.virtual_request_stack')->ignoreOnInvalid(),
```

Comment:
shouldn't we instead pass the request stack when the virtual one doesn't exist? that'd match the previous behavior

the solution might be to alias the virtual stack to the real one in FWB when not in debug, WDYT?

Reply:
I did that in a compiler pass now. Adding an alias is probably not a solution as long as we do not want to list older versions of the FrameworkBundle as a conflict.